great_expectations/expectations/metrics/map_metric.py

```diff
@@ -368,10 +380,19 @@ def inner_func(
                     metric_domain_kwargs, domain_type=domain_type
                 )
 
-                column_name = accessor_domain_kwargs["column"]
-                dialect = execution_engine.dialect_module
-                sqlalchemy_engine = execution_engine.engine
+                column_name: str = accessor_domain_kwargs["column"]
+
+                sqlalchemy_engine: sa.engine.Engine = execution_engine.engine
+                if not is_column_present_in_table(
+                    engine=sqlalchemy_engine,
+                    table_name=selectable.description,
+                    column_name=column_name,
+                ):
+                    raise ge_exceptions.ExecutionEngineError(
+                        f'Error: The column "{accessor_domain_kwargs.get("column")}" in BatchData does not exist.'
```

Comment:
Non-blocking suggestion since you already have the column_name:
```suggestion
                        f'Error: The column "{column_name}" in BatchData does not exist.'
```
(Here and in the other error messages).

Reply:
Fixed -- nice catch; thanks, @anthonyburdi !